test/mlabns/mlabns.cpp

```diff
@@ -34,6 +34,7 @@ TEST_CASE("Query can pass the settings to the dns level") {
     settings["mlabns/metro"] = "trn";
     settings["mlabns/policy"] = "random";
     settings["dns/nameserver"] = "8.8.8.1";
+    settings["dns/engine"] = "libevent";
```

Comment:
This is a specific test that tries to pass a **dns/nameserver** to the lowest level from the higher (mlab-ns -> http -> net -> dns) to see it the settings are passed correctly. Since we are trying to change the **dns/nameserver** options, that is not supported with the system resolver, I decided to use libevent for this specific case!